server/route.go

```diff
@@ -447,23 +446,15 @@ const (
 	RSID  = "RSID"
 	QRSID = "QRSID"
 
-	RSID_CID_INDEX   = 1
-	RSID_SID_INDEX   = 2
-	EXPECTED_MATCHES = 3
+	QRSID_PREFIX     = QRSID + ":"
```

Comment:
Based on latest discussion, let's get rid of `QRSID_PREFIX` and if you want, you could keep `QRSID_LEN = len(QRSID)`.